src/Lumi/Components/Table.purs

```diff
@@ -310,7 +310,27 @@ table = make component
       where
         selected = fromMaybe self.state.selected (toMaybe self.props.selected)
 
-        primaryColumn = toMaybe self.props.primaryColumn
+        primaryColumn = cleanUpNullables <$> toMaybe self.props.primaryColumn
+          where
+            cleanUpNullables
```

Comment:
Was there a particular reason not to write this as `cleanUpNullables r = r { getLink = toMaybe r.getLink }`?

Reply:
Ohh, I forgot you could change the record type if the labels have the same names, thanks